flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/optimize/StreamCommonSubGraphBasedOptimizer.scala

```diff
@@ -178,7 +178,7 @@ class StreamCommonSubGraphBasedOptimizer(planner: StreamPlanner)
 
       def getMiniBatchInterval: MiniBatchInterval = miniBatchInterval
 
-      override def needFinalTimeIndicatorConversion: Boolean = true
+      override def needFinalTimeIndicatorConversion: Boolean = false
```

Comment:
should return `isSinkBlock` here

Reply:
thanks for you response:) I've fixed failed test. 
How to rerun build.

Reply:
thanks for you response:) I've fixed failed test. 
How to rerun build.